src/plugins/dataflow/nodes/factories/live-output-rete-node-factory.tsx

```diff
@@ -72,7 +72,7 @@ export class LiveOutputReteNodeFactory extends DataflowReteNodeFactory {
     }
   }
 
-  private getNewValueForGrabber(num: number){
+  private getPercentageAsInt(num: number){
     if (num > 1)  return 100;
     if (num < 0)  return 0;
     return parseInt((num * 100).toFixed(2), 10);
```

Comment:
>    return parseInt((num * 100).toFixed(2), 10);

Out of curiosity, is there a reason to convert to string and then parse it? Seems like this should be the same as `Math.floor(num * 100)`, but `Math.round(num * 100)` might be preferable.

Reply:
@kswenson - good question.  The two expressions you listed above return the desired value just as well, so will include this change in my next PR: 

```ts
return Math.round(num * 100)
```

I am pretty sure that parseInt / toFixed circle is an artifact of my attempting to please the compiler and/or address errors,  thanks for pointing it out!